utility/test/account_test.go

```diff
@@ -35,8 +30,9 @@ func TestUtilityContext_AddAccountAmount(t *testing.T) {
 	require.NoError(t, err)
 
 	expected := initialAmount.Add(initialAmount, addAmount)
-	require.True(t, afterAmount.Cmp(expected) == 0, fmt.Sprintf("amounts are not equal, expected %v, got %v", initialAmount, afterAmount))
-	test_artifacts.CleanupTest(ctx) // TODO (team) need a golang specific solution for teardown
+	require.Equal(t, afterAmount, expected)
```

Comment:
You should reorder `expected` and `afterAmount` per the function interface: 

```go
func Equal(t [TestingT](https://pkg.go.dev/github.com/stretchr/testify/require#TestingT), expected interface{}, actual interface{}, msgAndArgs ...interface{})
```

The reason we want to do this is because the output to terminal will then show what was "expected" and what was "actual".

Ditto elsewhere in the code.

Source: https://pkg.go.dev/github.com/stretchr/testify/require#Equal